components/build/tekton-chains/chains-controller-deployment.yaml

```diff
@@ -8,7 +8,10 @@ spec:
     spec:
       containers:
       - name: tekton-chains-controller
-        image: ghcr.io/hacbs-contract/chains/controller:cd2d8c2c9fea3902248dd811913e7ea4fa73d639
```

Comment:
Could maybe leave it with a comment, e.g.:

      # Contract team fork. May be useful in future for testing pipeline run attestation:
      #image: ghcr.io/hacbs-contract/chains/controller@sha256...

Reply:
Yes! done.